format-specs/geoparquet.md

```diff
@@ -1,5 +1,51 @@
+# Geospatial Parquet format
 
-## A Geospatial Parquet format
+## Overview
 
-This page is a scratch pad for ideas to define a 'geo-parquet' format, aligning on a default schema and format for geospatial data in Parquet.
+The [Apache Parquet][parquet] provides a standardized open-source columnar storage format. This specification defines how geospatial data
+should be stored in parquet format, including the representation of geometries and the required additional metadata.
 
+## Geometry columns
+
+Geometry columns are stored using the `BYTE_ARRAY` parquet type. They are encoded as [WKB](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry#Well-known_binary).
+
+## Metadata
+
+geoparquet files include additional metadata at two levels:
+
+1. File metadata indicating things like the version of this specification used
+2. Column (chunk) metadata with additional metadata for each geometry column
+
+## File metadata
+
+All file-level metadata should be included under the "geoparquet" key in the parquet metadata.
+
+|     Field Name     |  Type  |                             Description                              |
+| ------------------ | ------ | -------------------------------------------------------------------- |
+| geoparquet_version | string | **REQUIRED** The version of the metadata standard used when writing. |
+| primary_column     | string | The name of the "primary" geometry column.                           |
+
+
+### Additional file metadata information
+
+**primary_column**: This indicates the "primary" or "active" geometry for systems that can store multiple geometries,
+but have a default geometry used for geospatial operations.
+
+## Column metadata
+
+Each geometry column should include additional metadata
```

Comment:
I'm just noticing that https://github.com/geopandas/geo-arrow-spec/pull/2 puts the per-column metadata in the file metadata under `geo.columns`, which is a mapping from column name to additional metadata.

@brendan-ward do you know why that uses file metadata, rather than putting it in the column metadata for each column? I suppose it's a bit easier for tools to parse when it's in the file metadata, since they don't have to check the metadata for each column?

Reply:
Additional note on this (I only realized this last week): a good reason we used file-level metadata and not field-level metadata, is that Arrow doesn't actually support reading/writing field-level metadata for Parquet. 

The Parquet spec has this, so this could be added to the Arrow implementation. But even then, this might be less ideal. In the Parquet spec this lives in the ColumnMetaData of a ColumnChunk of a RowGroup (see https://parquet.apache.org/documentation/latest/#metadata). So that would mean that the metadata is repeated for each RowGroup, and a single Parquet file can consist of many row groups.